ui/lib/core/addon/components/form-field.js

```diff
@@ -72,6 +72,9 @@ export default Component.extend({
     return '';
   }),
 
+  // This is only used for `optional-text` editType
+  showInput: false,
```

Comment:
Should this be added to the documented arguments at the top?

Reply:
Making a ticket for this.